client/reducers/all-users.js

```diff
@@ -0,0 +1,23 @@
+import {
+  RECEIVE_ALL_USERS,
+  LOG_OFF,
+  REQUEST_ALL_USERS
+} from '../actions/auth'
+
+const allUsers = (state = null, action) => {
```

Comment:
The default state should be `[]` instead of null. This is also true for the return statements below (in this reducer). We should also have a unit test for each of the actions this reducer manages.